android/app/src/androidTest/kotlin/coop/polypoly/polypod/CommunicationThroughPodApiWorks.kt

```diff
@@ -67,11 +67,11 @@ class CommunicationThroughPodApiWorks {
     @get:Rule
     val activityRule = ActivityScenarioRule(MainActivity::class.java)
 
-    @Test
+    // TODO: fetch instrumentation tests are disabled for now
+    // @Test
```

Comment:
I _think_ you could instead do it like this:

```
@Disabled("fetch instrumentation tests are disabled for now")
@Test
fun polyOut() {
...
```

I _think_ we would still see those tests as _disabled_ when we run all the tests, so we would see that there's something to do whenever we run the test suite - and wouldn't need a TODO comment.